Cpm.py

```diff
@@ -4,43 +4,98 @@
 class Activity:
 
     def __init__(self, name, duration):
-        self._name = name
-        self._duration = duration
-        self._depending_activities = []
+        self.name = name
+        self.duration = duration
+        self.min_time_to_start = 0
+        self.max_time_to_start = 0
+        #self._depending_activities = []
 
     def __str__(self):
-        print("Activity %s with duration %s", self._name, self._duration)
+        print("Activity %s with duration %s", self.name, self.duration)
+
+    def __eq__(self, other):
+        return self.name == other.name and self.duration == other.duration
+
+    def __ne__(self, other):
+        return not self == other
 
 
 class Project:
 
     def __init__(self, project):
+        # this method initializes a project object if no dictionary or None is given, an empty dictionary will be used
         if project is not None:
-            self._project = project
+            self.__project = project
+        else:
+            self.__project = {}
+
+        self.__project_duration = 0
```

Comment:
`__project_duration` should be calculated if there is a project content.

Reply:
i will midify it tomorrow

Reply:
done